keras_cv/layers/preprocessing/random_rotation.py

```diff
@@ -156,6 +156,9 @@ def augment_bounding_boxes(
         image_shape = tf.shape(image)
         h = image_shape[H_AXIS]
         w = image_shape[W_AXIS]
+        _, _, _, _, rest = tf.split(
```

Comment:
What is expected in `rest`?

Reply:
class_id, etc. It is kept open in case we add more values to it.